core/src/main/java/org/bitcoinj/core/AbstractBlockChain.java

```diff
@@ -589,7 +589,7 @@ private void connectBlock(final Block block, StoredBlock storedPrev, boolean exp
             throw new VerificationException("Block failed checkpoint lockin at " + (storedPrev.getHeight() + 1));
         if (shouldVerifyTransactions()) {
             for (Transaction tx : block.getTransactions())
-                if (!tx.isFinal(storedPrev.getHeight() + 1, block.getTimeSeconds()))
+                if (!tx.isFinal(storedPrev.getHeight() + 1, block.time().getEpochSecond()))
```

Comment:
Won't this need a rebase given a PR that I just approved.

Reply:
Sure, I'll rebase.